example/user/user.proto

```diff
@@ -80,4 +80,5 @@ message Task {
     };
     string name = 1;
     string description = 2;
+    int64 priority = 3;
```

Comment:
Are you manually including the field here to demonstrate that it works properly with an `int64` type in addition to just the automatic `int`?

Reply:
Yeah, exactly.